src/_premake_main.lua

```diff
@@ -230,7 +230,7 @@
 		-- script has run to allow for project-specific options
 		ok, err = p.option.validate(_OPTIONS)
 		if not ok then
-			print("Error: " .. err)
+			eprint("Error: " .. err)
```

Comment:
can't we just call p.error(...) here... 
it prints the error, and exits, which seems to be what is intended here anyway?

Reply:
Yes, I only added eprint() because I wasn't entirely sure if there was some reason I wasn't aware of that you were using print() instead of p.error().